radixdlt-java/src/main/java/com/radixdlt/client/core/network/RadixNetworkState.java

```diff
@@ -0,0 +1,23 @@
+package com.radixdlt.client.core.network;
+
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Current state in time of a {@link RadixNetwork}
+ */
+public class RadixNetworkState {
+	private final Map<RadixPeer, RadixPeerState> peers;
+
+	public RadixNetworkState(Map<RadixPeer, RadixPeerState> peers) {
```

Comment:
In future, using the builder pattern would probably be easiest way to construct this so you wouldn't need to pass maps around.

Reply:
But no worries for now.

Reply:
That's a good idea yeah. As you say, not necessary right now.